schema/common/defaults/configuration/opensearch.yml

```diff
@@ -0,0 +1,31 @@
+kind: configuration/opensearch
```

Comment:
Seems that not all the fields here line up or are present inside the validation:
epiphany/schema/common/validation/configuration/opensearch.yml